deployment/ansible/roles/driver.monit/tasks/main.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Download monit
+  get_url: >
+    url=https://mmonit.com/monit/dist/binary/{{ monit_version }}/monit-{{ monit_version }}-linux-x64.tar.gz
```

Comment:
Just wanted to ask why we'd use this vs. the version in the Ubuntu repositories?

```bash
vagrant@vagrant-ubuntu-trusty-64:~$ sudo apt-cache policy monit
monit:
  Installed: (none)
  Candidate: 1:5.6-2
  Version table:
     1:5.6-2 0
        500 http://archive.ubuntu.com/ubuntu/ trusty/universe amd64 Packages
```